app/views/account_reset/recovery_options/show.html.erb

```diff
@@ -0,0 +1,28 @@
+<% title t('account_reset.recovery_options.header') %>
+<%= render PageHeadingComponent.new.with_content(t('account_reset.recovery_options.header')) %>
+
+<p>
+  <%= t('account_reset.recovery_options.help_text') %>
+</p>
+
+  <%= render ProcessListComponent.new(heading_level: :h3, class: 'margin-y-3') do |c| %>
```

Comment:
This isn't within an `h2` heading section, so I don't think we should be assigning `heading_level` here. It's reported by aXe DevTools as an issue:

https://dequeuniversity.com/rules/axe/html/3.5/heading-order